lib/active_record_florder.rb

```diff
@@ -55,6 +55,14 @@ def step(value)
     def get_step
       @step || 2**16
     end
+
+    def populate(value)
+      @populate = value
+    end
+
+    def get_populate
```

Comment:
Well in fact this thing piss me off and I want refactor it. It's kind of ugly. I'm not much happy with all of that setter/getter code in here..